src/dialog-controller.js

```diff
@@ -76,11 +76,15 @@ export class DialogController {
             } else {
               this._reject(new DialogCancelError(output));
             }
-            return result;
+            return { wasCancelled: false };
```

Comment:
Did it for consistency, return type like `{wasCancelled: boolean} | void` seemed strange. Also I thought it will not be an overhead since it should be done once by controller.